### PR TITLE
website is missing link to github source #19

### LINKED ISSUE #19: website is missing link to github source
[showmyprs](https://showmyprs.com/) website is missing the link to github source.

### PR DESCRIPTION
**Pull request by cg-cnu**
Just added the link to the github source. This pr closes #19.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
   <span class="subheading">See all your Open Source contributions in one place</span>
   <input class="username js-username" type="text" autofocus id="username" placeholder="Enter your Github username" onkeypress="handleKeypress(event)"/>
   <button class="btn btn-primary" onclick="showPrs()">Show My Pr's</button>
-  <span class="madeBy">Made with ❤️ by <a href="https://twitter.com/geekykaran">geekykaran</a></span>
+  <span class="madeBy">Made with ❤️ by <a href="https://twitter.com/geekykaran">geekykaran</a>. You can find the source on <a href="https://github.com/karanjthakkar/showmyprs.com/">github</a></span>
```
Comment (karanjthakkar) [inline templates/index.html]: @cg-cnu Thanks so much for adding this. Definitely needed! 

Minor nits:
1. Change anchor text that says `github` -> `Github`
2. Remove trailing `/` from github link

Comment (cg-cnu) [reply templates/index.html]: Sure thing. on it 😄

**Comment by karanjthakkar** [comment]
@cg-cnu This change is live on the website: https://showmyprs.com Thanks! 🥇 💥

**Comment by cg-cnu** [comment]
Awesome! 👍

**Pull request merged**